### PR TITLE
feat: update deployment and value files

### PR DESCRIPTION
**Pull request by hugo-calado**
PR to add feature of custom livenessProbe and readinessProbe.

### CODE REVIEW

File: application/values.yaml [hunk 1]
```diff
@@ -40,6 +40,18 @@ service:
...
+  liveness:
+    enabled: true
+    probe: 
+      exec:
+        command:
+        - cat
+        - /tmp/healthy
+  readiness:
+    enabled: false
+    probe:
```
Comment (jfmrm) [inline application/values.yaml:53]: looks good, but disable probes by default and put these examples as comments.

**Review by jfmrm — CHANGES_REQUESTED** [review]

File: application/values.yaml (hunk 1 above)
Comment (hugocalad0) [reply application/values.yaml]: @jfmrm changes applied.

**Comment by jfmrm** [comment]
@hugo-calado looks good now, just 2 small things.
1. mind commit messages, they should be descriptive regarding the content of the PR and follow the pattern, this will be useful to maintain automatic version tagging soon.
2. have u tested the code? I mean run helm install and try to spin up a test application in our experiments cluster? Could u post some prints on the PR to show it's working?

**Comment by hugo-calado** [comment]
Finished validation of this PR.

Created specific namespace, then installed helm application:
image

For this validation, it was used the image and probes directly from kubernetes documentation (https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe).

Probes configured on values.yaml file:
image


Were able to validate that both liveness and readiness probes were applied successfully and were working as expected:
image

**Review by jfmrm — APPROVED** [review]

**Pull request merged**